lib/client.js

```diff
@@ -531,47 +538,15 @@ function Request(method, url) {
 }
 
 /**
- * Mixin `Emitter`.
+ * Mixin `Emitter` and `RequestBase`.
  */
 
 Emitter(Request.prototype);
```

Comment:
Maybe. The complication is the node class inherits from Stream, which inherits from EventEmitter so that's where the event emitter methods come from on the node side, vs the client side which uses component-emitter.

I was planning to just use `events.EventEmitter` and `util.inherits` and let browserify deal with that in the client, but not sure if that would be cleaner. Any idea if we really need the `"browser: {"emitter": "component-emitter"}` bit from `package.json` anymore? I'm not sure our test coverage regarding emitted events is that thorough though so I'm a bit cautious.